sha1-validation/Program.cs

```diff
@@ -0,0 +1,47 @@
+using System.Net.Security;
+using System.Security.Cryptography.X509Certificates;
+
+bool sha1RsaSignatureOnLastElementInChain = false;
+
+HttpClientHandler handler = new HttpClientHandler { 
+    CheckCertificateRevocationList = true,
+    ServerCertificateCustomValidationCallback = ServerCertificateCustomValidation, 
+};
+
+HttpClient client = new HttpClient(handler);
+
+try
+{
+    HttpResponseMessage response = await client.GetAsync("https://redhat.com");
+
+    string responseBody = await response.Content.ReadAsStringAsync();
+    Console.WriteLine($"{sha1RsaSignatureOnLastElementInChain}");
+    Console.WriteLine("PASS");
+    return 0;
+}
+catch (HttpRequestException e)
+{
+    Console.WriteLine("\nException Caught!");
+    Console.WriteLine("Message :{0} ", e.Message);
```

Comment:
```suggestion
    Console.WriteLine(e);
```

This will print out the full exception including the stacktrace.